### DISCUSSION TITLE
XMake 生成Qt QML VS项目编译运行，关闭UI界面会 Critical error detected c0000374

### DISCUSSION THREAD

**Discussion by mengxinxiaoqiang** [xmake-io/xmake#4246] [Q&A] [answered]
Qt SDK version ... 6.5.1
使用Vs 上的Qt Tools 生成的项目编译运行不会出现这个问题，百思不得其解，求各位大佬帮忙看看，感谢！！
![image](https://github.com/xmake-io/xmake/assets/56524631/097a1536-df29-4b0c-934c-b7cad8af6a63)
![image](https://github.com/xmake-io/xmake/assets/56524631/a0dc8f14-40da-42d8-98b7-620e4b8a217f)
![image](https://github.com/xmake-io/xmake/assets/56524631/3b02acf3-e52a-4920-925b-9be09beef38c)
![image](https://github.com/xmake-io/xmake/assets/56524631/60ac51ae-86d5-418d-af65-d5ca165fbae6)
![image](https://github.com/xmake-io/xmake/assets/56524631/068b3fba-b3b7-4777-b324-5897abd59fb0)
![image](https://github.com/xmake-io/xmake/assets/56524631/b1d8c693-c9e9-462f-8d0c-2b7b3aeefa24)
[TestProject.zip](https://github.com/xmake-io/xmake/files/12728322/TestProject.zip)

**Comment by mengxinxiaoqiang** [comment]
XMake 版本
![image](https://github.com/xmake-io/xmake/assets/56524631/065a7877-1e42-45d1-80f0-0b2a79d45356)

**Comment by waruqi** [comment, marked as answer]
切下 vs runtime 试试呢，看下 https://github.com/xmake-io/xmake/issues/4161

**Reply by mengxinxiaoqiang** [reply]
if is_mode("debug") then
    set_runtimes("MDd")
else
    set_runtimes("MD")
end
是的，这样就能解决，感谢大佬！
